### ISSUE TITLE
Is this project dead?

### ISSUE THREAD

**Issue by mar-ops** [SphtKr/homebridge-zway#136]
I've been using this plugin for a while, now just got a Fibaro "The button" and an Aeotec wallmote. I then realized that they are obviously not supported through the homebridge-zway plugin and that this plugin hasn't been updated in 3 years.

So I assume it's dead? Are there any alternatives? How difficult is it to add new device categories? In the read-me it says that in the trunk there is already something prepared for buttons/remotes. How can I use that?

**Comment by QuickSander** [comment]
I don't consider it dead, but you might want to check out some forks which are more active or solve your issue: https://techgaun.github.io/active-forks/index.html#SphtKr/homebridge-zway

**Comment by earlthesquirrel** [comment]
So, if not dead, why not merge in the fixes from all the other forks?  

Also some devices that are mapped into homekit show up as “not supported” which doesn’t make sense. ( basic switches and outlets ).  I’d start debugging but it’s hard to tell what’s what with all the forks and updates.

**Comment by QuickSander** [comment]
Agree, but since I have no control over that I switched to https://github.com/palasinio/homebridge-zway Later on I switched over to the Zway HomeKit Gateway you can install as an app in Zway server.

**Comment by earlthesquirrel** [comment]
Oh, mis-matched your handle with the owner.  I tried the Zway app, has the same problem -- devices showing up in Homekit as not supported.   Perhaps it's time for me to create a fork of forks...

**Comment by palasinio** [comment]
Would be nice to have this project alive.
@b5rt  managed to add a device https://github.com/palasinio/homebridge-zway/commit/cd5d578bdb7196482d8e2ed8f5a4319935083094
I just merged the pending pull requests and updated dependencies.
